app/views/movies/show.html.erb

```diff
@@ -1,34 +1,5 @@
-<h1><%= @movie.title %></h1>
```

Comment:
moved this to `views/movies/_details` partial

Reply:
Sounds good! Is that a standard / recommended approach? Ask as I don't use partials too often. Looks good.

Reply:
In this case, I wanted to avoid hitting any @movie methods because of how the Service returns an "error" message in the API response. The feedback I've gotten on partials is that they are generally best used for re-usable code but can also be used to abstract code and to make control flow in views more readable.